### ISSUE TITLE
CommsOutgoingPackets straight-up don't work

### ISSUE THREAD

**Issue by JordanLongstaff** [rjwut/ian#13]
My app sends hail messages to ally ships and status report requests to stations and the server offers no response whatsoever. CommsIncomingPackets seem to work just fine.

**Comment by JordanLongstaff** [comment]
Never mind, I got it working. I think the issue had something to do with the parsing of PlayerShip objects from ObjectUpdatePackets. There's an int in the payload that carries a ship's "capital ship ID", i.e. the ship it returns to if it's a fighter. Get this: if it's not a fighter, that int won't be in the payload, but the appropriate bit in the bitfield will still be on. This messes up the parser. I was able to fix this with an actual check on the player ship that verifies whether or not it's a fighter. Heck, this might have even been the cause of my "bad ObjectType" issue as well. And by the way, the order of the "capital ship ID" and "accent color" in the player ship parser aren't in line with the protocol docs.
